kloppy/helpers.py

```diff
@@ -129,23 +130,22 @@ def _frame_to_pandas_row_converter(frame: Frame) -> Dict:
         ball_owning_team=frame.ball_owning_team.value
         if frame.ball_owning_team
         else None,
-        ball_x=frame.ball_position.x if frame.ball_position else None,
-        ball_y=frame.ball_position.y if frame.ball_position else None,
+        ball_x=frame.ball_coordinates.x if frame.ball_coordinates else None,
+        ball_y=frame.ball_coordinates.y if frame.ball_coordinates else None,
     )
-    for jersey_no, position in frame.home_team_player_positions.items():
-        row.update(
-            {
-                f"player_home_{jersey_no}_x": position.x,
-                f"player_home_{jersey_no}_y": position.y,
-            }
-        )
-    for jersey_no, position in frame.away_team_player_positions.items():
-        row.update(
-            {
-                f"player_away_{jersey_no}_x": position.x,
-                f"player_away_{jersey_no}_y": position.y,
-            }
-        )
+    for player, coordinates in frame.players_coordinates.items():
+
+        if isinstance(player, Player):
```

Comment:
For what case is this check needed? I would assume it's always an Player instance

Reply:
This was before all data providers had Player as key in players_coordinates. Not it can be removed.